analysis/alignment/extract_matrix.py

```diff
@@ -2,6 +2,10 @@
 import sys
 import argparse
 import csv
+import sys
+
+csv.field_size_limit(sys.maxsize)
```

Comment:
maybe we should even do this which we already have in SWATHScoringReader.py

```
from sys import maxsize
import csv 

maxInt = maxsize
decrement = True

while decrement:
    # decrease the maxInt value by factor 10 
    # as long as the OverflowError occurs.
    # http://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072

    decrement = False
    try:
        csv.field_size_limit(maxInt)
    except OverflowError:
        maxInt = int(maxInt/10)
        decrement = True


```

that seems more portable: http://stackoverflow.com/questions/15063936/csv-error-field-larger-than-field-limit-131072